src/transformers/models/deberta/tokenization_deberta.py

```diff
@@ -434,11 +431,6 @@ def prepare_for_tokenization(self, text, is_split_into_words=False, **kwargs):
             text = " " + text
         return (text, kwargs)
 
-    # Copied from transformers.models.gpt2.tokenization_gpt2.GPT2Tokenizer._build_conversation_input_ids
-    def _build_conversation_input_ids(self, conversation: "Conversation") -> List[int]:
-        input_ids = []
-        for is_user, text in conversation.iter_texts():
-            input_ids.extend(self.encode(text, add_special_tokens=False) + [self.eos_token_id])
-        if len(input_ids) > self.model_max_length:
-            input_ids = input_ids[-self.model_max_length :]
-        return input_ids
+    @property
+    def default_chat_template(self):
+        return "{% for message in messages %}" "{{ message.content }}{{ eos_token }}" "{% endfor %}"
```

Comment:
I'm pretty sure this is here only because the tokenizer was copy-pasted from gpt2, DeBERTa cannot do generation AFAIK. So if you're removing the `# Copied from` statement, you can remove this too I think

Reply:
Removed!